app/views/layouts/_tracking_scripts.html.erb

```diff
@@ -1,18 +1,4 @@
-// End Rollbar Snippet
-</script>
-
```

Comment:
Ah nice, leveraging the Ruby gem's ability to integrate with JavaScript 💪 
https://docs.rollbar.com/docs/integration-with-rollbarjs